File: .sdk/tm/php/utility/MakeContext.php
<?php
declare(strict_types=1);

// SwissFederalRailwaysSbb SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class SwissFederalRailwaysSbbMakeContext
{
    public static function call(array $ctxmap, ?SwissFederalRailwaysSbbContext $basectx): SwissFederalRailwaysSbbContext
    {
        return new SwissFederalRailwaysSbbContext($ctxmap, $basectx);
    }
}

File: .sdk/tm/py/pkg/utility/make_context.py
# SwissFederalRailwaysSbb SDK utility: make_context

from projectname_sdk.core.context import SwissFederalRailwaysSbbContext


def make_context_util(ctxmap, basectx):
    return SwissFederalRailwaysSbbContext(ctxmap, basectx)
